Receipt/index.ts

```diff
@@ -17,4 +17,7 @@ export namespace Receipt {
 			isoly.Currency.is(value.currency)
 		)
 	}
+	export function validate(value: Receipt) {
+		return !!value.original && value.amount > 0 && value.vat >= 0 && isoly.Currency.is(value.currency)
```

Comment:
Can you change the interface to use `Amount` instead of both `currency` and `amount`? must be misstake from earlier. Should probably change `vat` to `Amount` as well.

Reply:
vat is a percentage that is represented as a number ex. 0.02